src/Emojis/emoji.php

```diff
@@ -30,5 +30,24 @@
     "kissing_smiling_eyes"      => "\u{1F619}",
     "kissing_closed_eyes"       => "\u{1F61A}",
     "relaxed"                   => "\u{263A}",
-
-];
\ No newline at end of file
+    "question"                  => "\u{2753}",
+    "worried"                   => "\u{1F61F}",
+    "cry"                       => "\u{1F622}",
+    "scream"                    => "\u{1F631}",
+    "pray"                      => "\u{1F64F}",
+    "thumbsup"                  => "\u{1F44D}",
+    "thumbsdown"                => "\u{1F44E}",
+    "shit"                      => "\u{1F4A9}",
+    "birthday"                  => "\u{1F382}",
+    "raised_hands"              => "\u{1F64C}",
+    "family"                    => "\u{1F46A}",
+    "kiss"                      => "\u{1F48B}",
+    "running"                   => "\u{1F3C3}",
+    "gun"                       => "\u{1F52B}",
+    "moneybag"                  => "\u{1F4B0}",
+    "computer"                  => "\u{1F4BB}",
+    "camera"                    => "\u{1F4F7}",
+    "house"                     => "\u{1F3E0}",
+    "office"                    => "\u{1F3E2}",
+    "white_check_mark"          => "\u{2705}"
```

Comment:
Thanks for this, but the unicode representation needs to follow an order from ascending to descending. For example, "\u{1F622}" needs to come before  before "\u{2753}"